### PR TITLE
Table V2 Documentation (State Handlers and Features)

### PR DESCRIPTION
**Pull request by matheusps**
#### What is the purpose of this pull request?

As the title says.
It also removes unused props.

#### What problem is this solving?

Great components become awesome with great documentation. 💅

#### How should this be manually tested?

Clone the repo and `yarn && yarn start`

#### Types of changes

- [ ] Bug fix (a non-breaking change which fixes an issue)
- [ ] New feature (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: react/components/EXPERIMENTAL_Table/README.md [hunk 1]
```diff
@@ -1,390 +1,325 @@
...
+    default:
+      return state
+  }
+}
 
-- The `name` should be rendered as simple as possible
-- The `email` header should contain 💌 emoji
-- Age should be rendered inside of a `Tag` component with de `bgColor` blue for underage heroes and pink for the older age.
-- The header title for `country` prop should be 'Nationality'.
+function SipleExample() {
```
Comment (thayannevls) [inline react/components/EXPERIMENTAL_Table/README.md]: this name is correct? 👀

Comment (emersonlaurentino) [inline react/components/EXPERIMENTAL_Table/README.md]:
```suggestion
function SimpleExample() {
```

File: react/components/EXPERIMENTAL_Table/README.md [hunk 2]
```diff
@@ -1,390 +1,325 @@
...
+        items={items}
+        onRowClick={onRowClick}
+        {...tableLoading}
+        empty={empty || customEmpty}
+        emptyState={empty ? emptyState : customEmptyState}
+      />
+    </>
   )
 }
+;<SipleExample />
```
Comment (emersonlaurentino) [inline react/components/EXPERIMENTAL_Table/README.md]:
```suggestion
;<SimpleExample />
```

File: react/components/EXPERIMENTAL_Table/README.md [hunk 3]
```diff
@@ -1,390 +1,325 @@
...
-  {
-    /** Definitions for the country prop */
-    id: 'country',
-    /** This means that the title shown for the country property will be Nationality */
-    title: 'Nationality',
-  },
-]
-```
+  const emptyState = {
+    label: 'This is an default empty state title',
```
Comment (emersonlaurentino) [inline react/components/EXPERIMENTAL_Table/README.md]:
```suggestion
    label: 'This is a default empty state for title',
```

File: react/components/EXPERIMENTAL_Table/README.md [hunk 4]
```diff
@@ -1,390 +1,325 @@
...
+  }
 
-##### Working example:
+  const customEmptyState = {
+    label: 'This is an default empty state title',
```
Comment (emersonlaurentino) [inline react/components/EXPERIMENTAL_Table/README.md]:
```suggestion
    label: 'This is a default empty state for title',
```

File: react/components/EXPERIMENTAL_Table/README.md [hunk 5]
```diff
@@ -1,390 +1,325 @@
...
-    age: 31,
-    country: '🇰🇪Wakanda',
-  },
-  {
-    id: 2,
-    name: 'Peter Parker',
-    email: 'spider.man@gmail.com',
-    age: 17,
-    country: '🇺🇸USA',
+    title: 'Email',
```
Comment (emersonlaurentino) [inline react/components/EXPERIMENTAL_Table/README.md]: don't you need an `id`?

File: react/components/EXPERIMENTAL_Table/README.md [hunk 6]
```diff
@@ -1,390 +1,325 @@
...
-    />
+    <>
+      <div className="mb5">
+        <ButtonGroup
+          buttons={ratios.map(ratio => (
+            <Button
+              size="small"
+              isActiveOfGroup={ratio.id === currentRatio.id}
+              onClick={() => setCurrentRatio(ratio)}>
+              Ratio: {ratio.title}
```
Comment (emersonlaurentino) [inline react/components/EXPERIMENTAL_Table/README.md]:
```suggestion
              {`Ratio: ${ratio.title}`}
```

**Pull request closed without merging**